### DISCUSSION TITLE
PDFで詳細を編集および入力する方法は？

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1825] [Q&A] [answered]
PDFで詳細を編集および入力する方法は？

画像を入力し、テキスト、テキストサイズ、テキストの色を揃えたい。そのためのチュートリアルやドキュメントはありますか？

![image](https://user-images.githubusercontent.com/49851974/177237836-93e0ed5d-73f3-4b45-ad11-f784118d0d18.png)

**Comment by kmuto** [comment, marked as answer]
最初に述べると、提示されているその1ページ目は、大扉・titlepage と呼ばれるもので、書籍においては表紙ではなく表紙をめくった後になるものです。書籍の制作においては、表紙と本文部は別々に制作します。

もし表紙を想定されていた場合は、review-init -wで用意した環境であればconfig-ebook.ymlという電子書籍用のファイルがあるはずですので、`REVIEW_CONFIG_FILE=config-ebook.yml rake pdf` としてみてください。
(ただ、Windowsだとこの設定方法がうまくいかないかもしれません。だめなときには `review-pdfmaker config-ebook.yml` を試してみてください。)
pdfmaker用の表紙ファイルはconfig.ymlのpdfmakerセクション下のcoverimageで設定しており、デフォルトではimagesフォルダ内のcover-a5.aiというAdobe Illustratorファイルが指定されています。これをほかの画像ファイルに置き換えることで表紙が変わります。

表紙ではなく、あくまでもこの大扉・ttilepageの見た目を変えたい(そのほか本文などもいろいろ変えたい)ということになると、Welcome to LaTeX World。Re:VIEWの範囲を超えており、LaTeX/TeXの知識を習得しsty/review-custom.styにオーバーライドや新規の記述をする必要があります。review-initにおいてreview-jsbookとreview-jlreqという選択がありましたが、review-jlreqのほうが比較すればカスタマイズしやすいです。しかしあくまでも比較しての範囲であり、事前に知識の習得が必要なことには変わりありませんし、TeXの記法や諸々の性質も考慮しないといけないので、CSSのようにすぐわかって感覚的というわけでもありません。

そのほかカスタマイズのヒントとしてはRe:VIEWナレッジベース https://review-knowledge-ja.readthedocs.io/ja/latest/ もご参照ください。
